Obsidian/Utilities/RentedArray.cs

```diff
@@ -27,13 +27,19 @@ namespace Obsidian.Utilities
         /// </summary>
         public Memory<T> Memory => array.AsMemory(0, Length);
 
+#nullable enable
+        /// <summary>
+        /// #nullable
+        /// </summary>
+        /// <param name="length"></param>
+        /// <param name="pool"></param>
```

Comment:
Unnecessary XMLdoc.

Reply:
Correct, this looks like I left some code here which should've been cleaned up.